app/assets/stylesheets/sections/app.scss

```diff
@@ -0,0 +1,39 @@
+#app-content {
+  padding-top: 75px;
+  min-height: 80%;
+}
+
+.user-logo {
+  display: block;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 150pt;
+  margin-top: 20px;
+
+}
+
+.powered-by {
+  margin-top: 0px;
```

Comment:
Properties should be ordered margin-bottom, margin-top<br>`0px` should be written without units as `0`